src/plugins/cachefilter/README.md

```diff
@@ -0,0 +1,16 @@
+- infos = Information about the cachefilter plugin is in keys below
+- infos/author = Marvin Mall <e1225943@student.tuwien.ac.at>
```

Comment:
I'll not use my personal email for this, that's for sure. My alias shall not be connected to my email in public.